diff --git a/facebook_tracking_pixel.info b/facebook_tracking_pixel.info
index 18d2865..5ee6a3a 100644
--- a/facebook_tracking_pixel.info
+++ b/facebook_tracking_pixel.info
@@ -3,4 +3,4 @@ description = Manage Facebook Tracking pixels
 core = 7.x
 files[] = tests/facebook_tracking_pixel.test
 package = Statistics
-configure = admin/config/system/facebooktrackingpixel
+configure = admin/config/system/facebook_tracking_pixel
